src/ssh/azext_ssh/_params.py

```diff
@@ -57,6 +59,7 @@ def load_arguments(self, _):
         c.argument('credentials_folder', options_list=['--keys-destination-folder', '--keys-dest-folder'],
                    help='Folder where new generated keys will be stored.')
         c.argument('port', options_list=['--port'], help='SSH Port')
+        c.argument('resource_tag', options_list=['--resource-tag'], help='Use port value from specified resource tag.')
```

Comment:
The help description on the config parameter is different than the vm parameter. Keep these the same since it has the same purpose in both commands